tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -531,7 +567,7 @@ def resume_test(host, port, test_ciphers, fips_mode, no_ticket=False):
 
         for async_result in results:
             if async_result.get() != 0:
-                failed = True 
+                failed = True
```

Comment:
Is the actual failure being lost here? Could each failure in the results list be stored, and raised in the IntegrationTestFailure exception?

We are trying to restructure all this, so I don't consider this blocking. It may help while debugging a failure in codebuild though.

Reply:
It could be. Another question in this section of code is whether we want to block on all results' competition or raise the error immediately. The change on this line is whitespace, so I haven't change the behavior. Definitely something to look into as part of integ tests enhancements.